src/components/Footer/Footer.scss

```diff
@@ -59,7 +56,7 @@
   }
 
   &--social {
-    svg {
+    &__icon {
```

Comment:
Is it because the linter want it to have class name?

Reply:
Yes, in the BEM article below it mentions that CSS should not use tag names or ids. It was throwing an error with the svg tag. Thus, I added the class to the SVG. 

Source: http://getbem.com/naming/